src/kw_ssh.sh

```diff
@@ -109,6 +122,91 @@ function run_command_in_the_remote()
   cmd_manager "$flag" "$ssh_compose"
 }
 
+# This function transfers a local file or directory to a
+# specified remote destination.
+function ssh_send_file_to_remote()
+{
+  local user
+  local port
+  local remote
+  local remote_file
+  local remote_file_host
+  local send_path
+  local to_path
+  local flag
+  local cmd
+  local send_option
+  local ssh_compose
+
+  flag=${flag:-'SILENT'}
+
+  user="${remote_parameters['REMOTE_USER']}"
+  port="${remote_parameters['REMOTE_PORT']}"
+  remote="${remote_parameters['REMOTE_IP']}"
+  remote_file="${remote_parameters['REMOTE_FILE']}"
+  remote_file_host="${remote_parameters['REMOTE_FILE_HOST']}"
+  send_path=${options_values['SEND']}
+  to_path=${options_values['TO']:-'.'}
+  flag=${options_values['VERBOSE']}
+
+  if [[ -f "$remote_file" ]]; then
+    cmd="scp -F ${remote_file} -r ${send_path} ${remote_file_host}:${to_path} > /dev/null 2>&1"
+  else
+    cmd="scp -P ${port} -r ${send_path} ${user}@${remote}:${to_path} > /dev/null 2>&1"
+  fi
+
+  cmd_manager "$flag" "$cmd"
+
+  if [[ $? -eq 0 ]]; then
+    success "File(s) uploaded successfully."
+  else
+    complain "An error occurred while uploading the file(s)."
+    exit 22 # EINVAL
+  fi
+}
+
+# This Function gets a file or directory from the
+# specified remote path and saves it to the local
+# machine at the specified path
+function ssh_get_file_from_remote()
+{
+  local user
+  local port
+  local remote
+  local remote_file
+  local remote_file_host
+  local get_path
+  local to_path
+  local flag
+  local cmd
+
+  flag=${flag:-'SILENT'}
+
+  user="${remote_parameters['REMOTE_USER']}"
+  port="${remote_parameters['REMOTE_PORT']}"
+  remote="${remote_parameters['REMOTE_IP']}"
+  remote_file="${remote_parameters['REMOTE_FILE']}"
+  remote_file_host="${remote_parameters['REMOTE_FILE_HOST']}"
+  get_path=${options_values['GET']}
+  to_path=${options_values['TO']:-'.'}
+  flag=${options_values['VERBOSE']}
+
+  if [[ -f "$remote_file" ]]; then
+    cmd="scp -F ${remote_file} -r ${remote_file_host}:${get_path} ${to_path} > /dev/null 2>&1"
```

Comment:
Same thing here, it could be useful to check beforehand if `get_path` exists.

Reply:
thx @magalilemes, i'm making theses changes. :)